### DISCUSSION TITLE
HTML files render but cannot run preview

### DISCUSSION THREAD

**Discussion by craaaa** [quarto-dev/quarto-cli#3702] [Q&A]
I'm running Quarto downloaded via Homebrew (Mac v1.2.280).

When I run `quarto preview [file.qmd]` or click "Render" in VSCode or RStudio, my .qmd file renders into html files but the preview fails with the following error:

```
#omitted rendering stuff 
Output created: hello.html

Preparing to preview

Watching files for changes
ERROR: NotFound: No such file or directory (os error 2)
```

I've tried running preview on command line, from VSCode, and in RStudio, but they have all failed. I've been using pretty generic files: `hello.qmd` from the [tutorial](https://quarto.org/docs/get-started/hello/text-editor.html) and the default file after opening a new Quarto document in RStudio, and rendering works fine, so I'm not sure what I'm doing wrong. Any help would be appreciated!

**Comment by dragonstyle** [comment]
We've had some reports like this related to network file systems or other non-default file system configurations (e.g.   a dropbox or google drive folder). Is anything like that in play for you? 

We're collecting all the issues related to differing file systems configurations to see if we can better understand why this is happening and what we can do to resolve it...
